backend/endpoints/identity.py

```diff
@@ -124,7 +124,7 @@ def __init__(
         self.avatar = avatar
 
 
-@protected_route(router.put, "/users/{user_id}", ["users.write"])
+@protected_route(router.patch, "/users/{user_id}", ["users.write"])
```

Comment:
Double-check that this stilll works case put and patch expect different data methods (body, form or query params)

Reply:
yes! it still works, changed to unify with `update_rom` endpoint